examples/apis/invoice/get/main.go

```diff
@@ -0,0 +1,28 @@
+package get
```

Comment:
nomear como main, assim como os outros exemplos
```suggestion
package main
```